app/backend/src/couchers/utils.py

```diff
@@ -96,6 +97,19 @@ def get_coordinates(geom):
     return (shp.y, shp.x)
 
 
+def to_timezone(lat, lng):
+    """
+    Returns the timezone name corresponding the lat, lng input, or None if the input is
+    The timezone names are defined in this timezone database:
+    https://www.iana.org/time-zones
+    """
+    tf = TimezoneFinder(in_memory=True)
+    try:
+        return tf.timezone_at(lat=lat, lng=lng)
+    except ValueError as ve:
+        print("Error: to_timezone():", ve)
```

Comment:
This seems incomplete.

Does TF not have its own exception we could catch?

Reply:
I followed the sequence of methods and functions that are called, and only a ValueError can be raised.